docs/source/index.rst

```diff
@@ -97,5 +96,7 @@ with more content coming soon.
    api_ref_intro
    api_ref_quantization
    api_ref_dtypes
+   serialization
+   
```

Comment:
you need to create a second caption for docs cause tutorial is not an API reference

Reply:
this is also not a tutorial, should I add "Concepts"?